### ISSUE TITLE
this is fake plugin, don't install it!!!!!!!!!!!!

### ISSUE THREAD

**Issue by chenghuili** [yireo/Yireo_Webp2#75]
this is fake plugin, once you installed it, you couldn't remove it, if you remove it, your website would be broken.
don't install it!!!!!!!!!!!!

**Comment by jissereitsma** [comment]
Haha, thanks for contributing to open source. I'm not sure why you call this free open source tool that I've spent weeks of labor on "fake".

Perhaps because you are bumping into a technical issue that you are able to solve yourself? Please note that a website is broken most likely because of an error. If you are unable to troubleshoot errors like these, it is basically your own fault that you are installing something in a Magento application that you are unable to manage properly. Those errors are there to help you address the issue. But instead of searching for that error, you have chosen to bash me personally in the face by calling this extension is "fake".

Sorry to hear that you are using free stuff, not putting the fair amount of time to analyse why your Magento instance is completely different from all those 100s other shops that are using this extension successfully, and even sorrier to see you get angry at me and this extension, instead of gently asking for some honest help.

In short, open source does not work for you in general. I would advice you to stick to paid extensions, so you can pay to smash the creators in the face.

Kay, thanks, bye.
